<?xml version="1.0"?>
<launch>

    <param name="total_uav" value="1"/>

    <node pkg="sensor_fusion" type="target.py" name="target" required="True" output="screen">
	</node>
    
    <group ns = "uav1">
        <include file="$(find sensor_fusion)/launch/uav.launch" >
            <arg name="ID" value="1"/>
        </include>
    </group>


</launch>
    
   